adagrams/game.py

```diff
@@ -1,11 +1,151 @@
+import random
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+    #Create a set of 10 letters that matches the letter frequency distribution#
+    draw = []
+    letter_pool_copy = LETTER_POOL.copy()
+    letters_list = list(letter_pool_copy.keys())
+
+    while len(draw) < 10:       
+        #Take a random number within the len of letter pool and get the letter in that index
+        random_letter = letters_list[random.randint(0,len(letter_pool_copy)-1)]
```

Comment:
Right now your code isn't actually utilizing the weight distribution of the letters. Currently, all letters have the same chance of being selected (they all can only appear their respective limits). `randint` is only randomly selecting from a pool of integers from `0` to `26`. This means that `Z` is just as likely as likely to be selected as `E`. You would need it to select from a pool of integers from `0` to `98` to have the correct distribution.  How could we change your letter pool to consider this weighted distribution?

Reply:
Thank you!! It looks like I forgot my statistics classes. Fixed! 🎉